core/src/main/resources/jenkins/model/Jenkins/login_tr.properties

```diff
@@ -25,3 +25,7 @@ User=Kullan\u0131c\u0131
 Password=\u015eifre
 Remember\ me\ on\ this\ computer=Beni bu bilgisayarda hat\u0131rla
 login=giri\u015f
+Invalid\ username\ or\ password=Kullan\u0131c\u0131 ad\u0131 veya \u015Fifre yanl\u0131\u015F
```

Comment:
Aslında password'un asıl karşılığı 'parola', çok yaygın bir hata bu.

Reply:
It's translated as "şifre" in all other places in Jenkins and it's not in scope of this PR to change that. It should be changed in a follow up/dedicated PR.